air/tests/test_module/features/data_merging/data_merge.rs

```diff
@@ -219,6 +219,7 @@ fn stream_merge() {
 }
 
 #[test]
+#[ignore]
```

Comment:
It worth clarifying this change (as well as below).

Reply:
Otherwise it will certainly be ignored forever.

Reply:
Mike had suggested this.